### PR TITLE
Make markers clickable and add a popover bubble with title/offset

### PR DESCRIPTION
**Pull request by bkeese**
Fixes #2167, with changes in avalon repo, branch feature/#2167-markers.

### CODE REVIEW

**Review by cjcolvar — APPROVED** [review]
:+1: Looks good to me.
@adamjarling thoughts?

File: app/assets/stylesheets/me-add-to-playlist.css.scss.erb [hunk 1]
```diff
@@ -1,7 +1,49 @@
...
+  background: #eee;
+  height: auto !important;
+  width: 100px !important;
+  border: solid 1px #333;
+  top: auto;
+  bottom: 8px;
+  transform: translate(-50%, 0);
+  text-align: center;
+  color: #111;
+  z-index: 1;
```
Comment (adamjarling) [inline app/assets/stylesheets/me-add-to-playlist.css.scss.erb:21]: As long as this z-index property is doing what it's intended, all good.  Generally for a z-index to take effect the class would need a `position` property, but perhaps it's inherited.

**Review by adamjarling — APPROVED** [review]
Looks like some pretty fine-tuned CSS @bkeese Nice work.

File: app/assets/stylesheets/me-add-to-playlist.css.scss.erb (hunk 1 above)
Comment (bkeese) [reply app/assets/stylesheets/me-add-to-playlist.css.scss.erb:21]: Thanks. position is inherited.

**Pull request merged**